packages/@aws-cdk/aws-rds/README.md

```diff
@@ -13,86 +14,126 @@
-    masterUsername: 'syscdk',
-    vpc
+const instance = new rds.DatabaseInstance(this, 'Instance', {
+  engine: rds.DatabaseInstanceEngine.ORACLE_SE1,
+  // optional, defaults to the engine's default instance type
+  instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+  masterUsername: 'syscdk',
+  vpc,
 });
 ```
+
 By default, the master password will be generated and stored in AWS Secrets Manager.
 
+To use a specific version of the engine
+(which is recommended, in order to avoid surprise updates when RDS add support for a newer version of the engine),
+use the static factory methods on `DatabaseInstancecEngine`:
+
+```typescript
+const instance = new rds.DatabaseInstance(this, 'Instance', {
+  engine: rds.DatabaseInstanceEngine.oracleStandardEdition1({
+    engineVersion: '19.0.0.0',
+  }),
+  ...
+});
+```
+
 To use the storage auto scaling option of RDS you can specify the maximum allocated storage.
 This is the upper limit to which RDS can automatically scale the storage. More info can be found
 [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIOPS.StorageTypes.html#USER_PIOPS.Autoscaling)
 Example for max storage configuration:
 
 ```ts
-const instance = new DatabaseInstance(stack, 'Instance', {
-    engine: rds.DatabaseInstanceEngine.ORACLE_SE1,
-    instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
-    masterUsername: 'syscdk',
-    vpc,
-    maxAllocatedStorage: 200
+const instance = new rds.DatabaseInstance(this, 'Instance', {
+  engine: rds.DatabaseInstanceEngine.ORACLE_SE1,       
+  // optional, defaults to the engine's default instance type
+  instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+  masterUsername: 'syscdk',
+  vpc,
+  maxAllocatedStorage: 200,
 });
 ```
 
 Use `DatabaseInstanceFromSnapshot` and `DatabaseInstanceReadReplica` to create an instance from snapshot or
 a source database respectively:
 
 ```ts
-new DatabaseInstanceFromSnapshot(stack, 'Instance', {
-    snapshotIdentifier: 'my-snapshot',
-    engine: rds.DatabaseInstanceEngine.POSTGRES,
-    instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.LARGE),
-    vpc
+new rds.DatabaseInstanceFromSnapshot(stack, 'Instance', {
+  snapshotIdentifier: 'my-snapshot',
+  engine: rds.DatabaseInstanceEngine.POSTGRES,     
+  // optional, defaults to the engine's default instance type
+  instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.LARGE),
+  vpc,
 });
 
-new DatabaseInstanceReadReplica(stack, 'ReadReplica', {
-    sourceDatabaseInstance: sourceInstance,
-    instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.LARGE),
-    vpc
+new rds.DatabaseInstanceReadReplica(stack, 'ReadReplica', {
+  sourceDatabaseInstance: sourceInstance,
+  instanceType: rds.DatabaseInstanceEngine.POSTGRES.defaultInstanceType,
```

Comment:
Why do we have default instance types per engine?

Reply:
I'm not sure I understand your question here.

Reply:
Same as this one - https://github.com/aws/aws-cdk/pull/8686#discussion_r444801448

Keep the defaults internal to the users of RDS.